test/index.test.ts

```diff
@@ -1,31 +1,50 @@
 import { strictEqual, deepStrictEqual } from "assert";
 import { describe, test } from "mocha";
-import { getAccounts, getConnection } from "@tableland/local";
+import { getAccounts, getDatabase, getValidator } from "@tableland/local";
 
 describe("index", function () {
   // Note that we're using the second account here
   const [, signer] = getAccounts();
-  const sdk = getConnection(signer);
+  const db = getDatabase(signer);
+  const validator = getValidator();
 
   test("create", async function () {
-    const { name } = await sdk.create("counter integer", { prefix: "table" });
-    strictEqual(name, "table_31337_2");
+    const { meta } = await db
+      .prepare("CREATE TABLE my_table (counter integer);")
+      .all();
+    strictEqual(meta.txn?.name, "my_table_31337_2");
   });
 
   test("insert", async function () {
-    const { hash } = await sdk.write("insert into table_31337_2 values (1);");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("insert into my_table_31337_2 values (1);")
+      .all();
+
+    strictEqual(typeof meta.txn?.transactionHash, "string");
+    strictEqual(typeof meta.txn?.chainId, "number");
+
+    const txnReceipt = await validator.receiptByTransactionHash({
+      transactionHash: meta.txn?.transactionHash,
+      chainId: meta.txn?.chainId,
+    });
     strictEqual(txnReceipt?.chainId, 31337);
   });
 
   test("update", async function () {
-    const { hash } = await sdk.write("update table_31337_2 set counter=2;");
-    const txnReceipt = await sdk.receipt(hash);
+    const { meta } = await db
+      .prepare("update my_table_31337_2 set counter=2;")
+      .all();
+    const txnReceipt = await validator.receiptByTransactionHash({
+      transactionHash: meta.txn?.transactionHash as string,
+      chainId: meta.txn?.chainId as number,
```

Comment:
using the `as` type assertion or a ts-ignore comment does work, but is potentially not good practice.  It would be good if these tests could serve as example for best practices.

Reply:
Yeh that shouldn't be required. Did something funny happen with the latest release in terms of types?